### PR TITLE
Deleted unused class LogRecorder

### LINKED ISSUE #130: Remove unused LogRecorder class in tests

### PR DESCRIPTION
**Pull request by Coldplayer1995**
Closes #130

### CODE REVIEW

File: test/__init__.py [hunk 1]
```diff
@@ -228,28 +228,3 @@ def __init__(self):
 
     def emit(self, record):
         self.records.append(record)
-
-
-class LogRecorder(object):
-    def __init__(self, target=logging.root):
-        super(LogRecorder, self).__init__()
-        self._target = target
-        self._handler = _ListHandler()
```
Comment (pquentin) [inline test/__init__.py]: Please remove `_ListHandler` too (the class just above)

Comment (Coldplayer1995) [reply test/__init__.py]: done

**Comment by pquentin** [comment]
@Coldplayer1995 anything I can do to help?

**Comment by Coldplayer1995** [comment]
I will try to do that first thing when I come back home today :)

**Comment by Coldplayer1995** [comment]
I did install the nox on my machine unfortunatelly there seems to be some problems with Env var name on my windows machine will investigate further.

**Comment by pquentin** [comment]
Oh, sorry to hear that. Another option is to just remove the logging import and let the CI tell us if that was enough. :)

**Comment by pquentin** [comment]
@Coldplayer1995 ping! Do you still want to work on this?

**Comment by pquentin** [comment]
Closing to let someone else work on it. Thanks!

**Pull request closed without merging**